### PR TITLE
Change to new running notes db for ONT FCs and projects

### PR DESCRIPTION
**Pull request by aanil**
* New partitioned running notes db to eventually separate running notes from project/flowcell/workset docs.
* Start off using it for just ONT flowcells and projects. 
    - Some temp variables included to select these, which have to be removed when the migration is complete.
* Dependent on https://github.com/SciLifeLab/StatusDB_views/pull/149

### CODE REVIEW

File: status/running_notes.py [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+                    application,
+                    [proj_coord],
+                    proj_ids,
+                    note,
+                    categories,
+                    user,
+                    created_time,
+                    "creation",
+                )
+        if note_type in ["flowcell", "workset", "flowcell_ont"]:
```
Comment (alneberg) [inline status/running_notes.py:168]: Do we not notify users for these type of running notes?

File: run_dir/design/project_samples.html [hunk 2]
```diff
@@ -569,6 +569,6 @@ <h3 class="modal-title">Caliper Image</h3>
 <script src="/static/js/jquery.dataTables.min.js"></script>
 <script src="/static/js/clipboard.min.js"></script>
 <script src="/static/js/links.js?v={{ gs_globals['git_commit'] }}" id="ln-js" data-workset="{{ project }}"></script>
-<script src="/static/js/running_notes.js?v={{ gs_globals['git_commit'] }}" id="rn-js" data-project="{{project}}" data-is-ont="{{ is_ont_proj }}"></script>
+<script src="/static/js/running_notes.js?v={{ gs_globals['git_commit'] }}" id="rn-js" data-project="{{project}}" data-project="{{project}}""></script>
```
Comment (alneberg) [inline run_dir/design/project_samples.html:572]: `data-project` is now duplicated

**Review by alneberg — APPROVED** [review]
Thank you! Looks good!

**Comment by aanil** [comment]
Merging with approval.

**Pull request merged**